org.tropy.Tropy.yml

```diff
@@ -0,0 +1,186 @@
+    config-opts:
+      - --disable-dec265
+      - --disable-sherlock265
+    cleanup:
+      - /bin
+    sources:
+      - type: archive
+        url: https://github.com/strukturag/libde265/releases/download/v1.0.12/libde265-1.0.12.tar.gz
+        sha256: 62185ea2182e68cf68bba20cc6eb4c287407b509cf0a827d7ddb75614db77b5c
+  - name: libheif
+    builddir: true
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DENABLE_PLUGIN_LOADING=OFF
+      - -DWITH_EXAMPLES=OFF
+      - -DWITH_LIBDE265=ON
+      - -DWITH_X265=OFF
+    sources:
+      - type: archive
+        url: https://github.com/strukturag/libheif/releases/download/v1.16.2/libheif-1.16.2.tar.gz
+        sha256: 7f97e4205c0bd9f9b8560536c8bd2e841d1c9a6d610401eb3eb87ed9cdfe78ea
+
+  - name: cgif
+    builddir: true
+    buildsystem: meson
+    config-opts:
+      - --buildtype=release
+    sources:
+      - type: archive
+        url: https://github.com/dloebl/cgif/archive/V0.3.2.tar.gz
+        sha256: 0abf83b7617f4793d9ab3a4d581f4e8d7548b56a29e3f95b0505f842cbfd7f95
+
+  - name: poppler-data
+    builddir: true
+    buildsystem: cmake-ninja
+    sources:
+      - type: archive
+        url: https://poppler.freedesktop.org/poppler-data-0.4.12.tar.gz
+        sha256: c835b640a40ce357e1b83666aabd95edffa24ddddd49b8daff63adb851cdab74
+  - name: poppler-glib
+    builddir: true
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DENABLE_ZLIB=ON
+      - -DENABLE_LIBTIFF=ON
+      - -DENABLE_LIBPNG=ON
+      - -DENABLE_GLIB=ON
+      - -DENABLE_LIBOPENJPEG=openjpeg2
+      - -DENABLE_DCTDECODER=libjpeg
+      - -DENABLE_BOOST=OFF
+      - -DENABLE_CPP=OFF
+      - -DENABLE_UTILS=OFF
+      - -DENABLE_QT5=OFF
+      - -DENABLE_QT6=OFF
+    cleanup:
+      - /lib/girepository-1.0
+    sources:
+      - type: archive
+        url: https://poppler.freedesktop.org/poppler-23.10.0.tar.xz
+        sha256: 31a3dfdea79f4922402d313737415a44d44dc14d6b317f959a77c5bba0647dd9
+
+  - name: libvips
+    builddir: true
+    buildsystem: meson
+    config-opts:
+      - --buildtype=release
+      - -Ddeprecated=false
+      - -Dintrospection=false
+      - -Dmodules=disabled
+      - -Dppm=false
+      - -Danalyze=false
+      - -Dradiance=false
+    cleanup:
+      - /bin/vips*
+    sources:
+      - type: archive
+        url: https://github.com/libvips/libvips/releases/download/v8.14.5/vips-8.14.5.tar.xz
+        sha256: 90374e9f6fbd5657b5faf306cacda20658d6144d385316b59b865bc1a487b68d
+
+  - name: tropy
+    buildsystem: simple
+    build-options:
+      append-path: /usr/lib/sdk/node18/bin
+    sources:
+      - type: archive
+        url: https://github.com/tropy/tropy/archive/refs/tags/v1.15.1.tar.gz
+        sha256: e4d1b7a36f5cc0c7cd1621b713d7aeb6a6988aa935d4e4d34a488789c85a9e62
+        dest: tropy
+      - type: file
+        path: tropy.sh
+    build-commands:
+      - install -Dm755 tropy.sh /app/bin/tropy
+      - cd tropy
+      - npm install --no-progress --no-audit --no-fund
```

Comment:
where does it get the packages?

Probably need to use https://github.com/flatpak/flatpak-builder-tools to generate the list.

Reply:
Ah, I see thanks!